ci-operator/step-registry/hypershift/aws/e2e/hypershift-aws-e2e-workflow.yaml

```diff
@@ -0,0 +1,36 @@
+workflow:
+  as: hypershift-aws-e2e
+  documentation: |-
+    The HyperShift aws e2e workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in the `osd-hypershift` AWS account and is destroyed after tests
+    complete.
+
+    The HyperShift launch capability is currently supported by the HyperShift
+    team. For now, please direct all questions and comments to:
+
+    - Alberto Lamela (agarcial@redhat.com)
+    - Seth Jennings (sjenning@redhat.com)
+    - Alvaro Aleman (alvaroaleman@redhat.com)
+    - Cesar Wong (cewong@redhat.com)
+
+    Learn more about HyperShift here: https://github.com/openshift/hypershift
+
+    Track HyperShift's development here: https://issues.redhat.com/projects/HOSTEDCP
+  steps:
+    post:
+    - chain: hypershift-dump
```

Comment:
It doesn't look like we need this step, since we should dump in the test. The `hypershift-aws-destroy-nested-management-cluster` chain already dumps the management cluster.

Reply:
Right, dropped it. Note: running this here would solve the problem of our e2e not dumping stuff when the overall ci timeout is reached. I dropped it for now and we can reevaluate the above a consistent approach in the future.